akka-operator/values.yaml

```diff
@@ -43,3 +43,8 @@ provider:
 gcp:
   repository: gcr.io/cloud-marketplace/lightbend-public/akka-cloud-platform
   deployer: false
+# Commercial image credentials - set `username` and `password` to install as Kubernetes Secret.
+imageCredentials:
+  username: false
+  password: false
+  registry: commercial-registry.lightbend.com
```

Comment:
I copy/pasted from https://github.com/lightbend/console-charts/blob/f7fd66b406/enterprise-suite/values.yaml#L84-L87 without giving it too much thought.

But now that you mention it, I think keeping the registry next to the credentials makes sense since these credentials are only useful on that registry but I should change the object name:

```
gcp:
  repository: gcr.io/cloud-marketplace/lightbend-public/akka-cloud-platform
# Commercial image credentials - set `username` and `password` to install as Kubernetes Secret.
lightbendSubscription:
  username: false
  password: false
  registry: commercial-registry.lightbend.com
```

Reply:
ok, so, at least rename it to `lightbendSubscription`.
Not fussy about it, but I still think that separating the registry from the credentials is a good thing for several reasons:

- air-gapped installations ppl will have something like:

 ```yaml
 lightbendCredentials: {}
 image.registry: `my-proxy-to-commercial-registry`
 ```
 - entire object re-definition is cleaner, without having to wonder how objects are merged:
 
 ```yaml
 imageCredentials:
   username: my-username
   password: my-password
 ```
 
It would be nice to have another opinion on the subject.

Reply:
air-gapped people can override the repository using `-set image.repository` and manually provide their own k8s `Secret` and the use `-set imagePullSecrets=[my-pull-secret]`. 

I wanted to keep all the Lightbend repository in a single place so it's clear this goes together.